examples/jsm/nodes/loaders/NodeMaterialLoader.js

```diff
@@ -1,5 +1,27 @@
 import { MaterialLoader } from 'three';
```

Comment:
The file should import materials from nodes/materials/Materials.js.

Reply:
good catch!